builders/integrations/indexers/subsquid.md

```diff
@@ -88,12 +94,55 @@ To get started indexing Substrate data on Moonbeam, you'll need to create a Subs
         const processor = new SubstrateBatchProcessor();
         processor.setDataSource({
           chain: '{{ networks.moonbase.rpc_url }}',
-          // Resolves to 'https://moonbase.archive.subsquid.io'
-          archive: lookupArchive('moonbase', { type: 'Substrate' }),
-        });
+          // Resolves to 'https://v2.archive.subsquid.io/network/moonbase-testnet'
+          archive: lookupArchive('moonbase', {type: 'Substrate', release: 'ArrowSquid'}),
+        })
+
         ```
 
-And that's all you have to do to configure your Subsquid project to index Substrate data on Moonbeam! Now you can update the `schema.graphql`, `typgen.json`, and `src/processor.ts` files to index the data you need for your project!
+5. There's one more quick change to make to the template. The Subsquid Substrate template is configured to process Substrate account types, but Moonbeam uses Ethereum-style accounts. In the `getTransferEvents` function, remove the ss58 encoding of the `from` and `to` fields. In an unmodified Substrate template, the `from` and `to` fields are ss58 encoded as shown:
```

Comment:
Can you please specify what file the `getTransferEvents` function is in (make sure to wrap it in back ticks please)?